### PR TITLE
Custom banner css

### PR DESCRIPTION
**Pull request by scotthasbrouck**

### CODE REVIEW

File: src/3_branch.js [hunk 1]
```diff
@@ -1255,7 +1258,7 @@ if (WEB_BUILD) { // jshint undef:false
 			icon: options['icon'] || '',
 			title: options['title'] || '',
 			description: options['description'] || '',
-			openAppButtonText: options['openAppButtonText'] || 'View in app',
+			openAppButtonText: options['openAppButtsonText'] || 'View in app',
```
Comment (dmitrig01) [inline src/3_branch.js]: typo

Comment (scotthasbrouck) [inline src/3_branch.js]: oops

**Comment by scotthasbrouck** [comment]
Fixed @dmitrig01 lmk if this looks ready to release

**Comment by dmitrig01** [comment]
great

**Pull request merged**